nimbus/src/evaluator.rs

```diff
@@ -47,9 +47,39 @@ pub fn evaluate_enrollment(
     app_context: &AppContext,
     exp: &Experiment,
 ) -> Result<ExperimentEnrollment> {
-    // Verify the application-id matches the application being targeted
+    // Verify the app_name matches the application being targeted
     // by the experiment.
```

Comment:
Perhaps both of these match behaviors want one or more unit tests?

Reply:
Good point, the current tests really only exercise the `app_id` part of this.  I'll add some better coverage for the app name and channel.